src/object.d

```diff
@@ -4178,3 +4073,108 @@ void __ArrayDtor(T)(T[] a)
     foreach_reverse (ref T e; a)
         e.__xdtor();
 }
+
```

Comment:
I wonder if it makes sense to add a big comment along the lines of:
```D
//******************
//* Runtime Hooks: *
//******************
```

To better designate the file structure.